docs/3-building-blocks-of-oop-part-2/hometask/src/Weapon.ts

```diff
@@ -1 +1,67 @@
-// your code goes here
\ No newline at end of file
+import { Item } from "./Item";
+
+export class Weapon extends Item {
+  MODIFIER_CHANGE_RATE: number = 0.05;
+  baseDamage: number;
+  baseDurability: number;
+  damageModifier: number = 0;
+  durabilityModifier: number = 0;
+  effectiveDamage: number;
+  effectiveDurability: number;
+
+  constructor(
+    name: string,
+    baseDamage: number,
+    baseDurability: number,
+    value: number,
+    weight: number
+  ) {
+    super(name, value, weight);
+
+    this.baseDamage = baseDamage;
+    this.baseDurability = baseDurability;
+    this.effectiveDamage = this.baseDamage + this.damageModifier;
+    this.effectiveDurability = this.baseDurability + this.durabilityModifier;
+  }
```

Comment:
let's move lines 23-24 to the methods `getDamage` and `getDurability`

Reply:
done